### PR TITLE
i18n(es): Updated `view-transitions.mdx` & `data-fetching.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `view-transitions.mdx & `data-fetching.mdx` based on these commits f85cb23b8fb09804b7ed6edde5e17e93b9a283fb, ede20900ca3368c37a6fb61cd7fe794181a9b484 & 1f3c047522c85e3978c8b1538059e3345a1a0e7e

### CODE REVIEW

File: src/content/docs/es/guides/data-fetching.mdx [hunk 1]
```diff
@@ -8,7 +8,9 @@ Los archivos `.astro` pueden hacer fetching de datos al momento de la compilaci
 
 ## `fetch()` en Astro
 
-Todos los [componentes de Astro](/es/core-concepts/astro-components/) tienen acceso a la función global [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/fetch) en el script del componente para realizar solicitudes HTTP a las APIs. Esta llamada se ejecutará en el momento de la compilación y los datos estarán disponibles en tu componente para generar HTML dinámico. Si el modo [SSR](/es/guides/server-side-rendering/) esta habilitado, cualquier llamada fetch se ejecutará en el tiempo de ejecución.
+Todos los [componentes de Astro](/es/core-concepts/astro-components/) tienen acceso a la [función global `fetch()`](https://developer.mozilla.org/es/docs/Web/API/fetch) en su script de componente para hacer peticiones HTTP a APIs usando la URL completa (e.g. https://example.com/api o `Astro.url + "/api"`).
```
Comment (Waxer59) [inline src/content/docs/es/guides/data-fetching.mdx:11]:
```suggestion
Todos los [componentes de Astro](/es/core-concepts/astro-components/) tienen acceso a la [función global `fetch()`](https://developer.mozilla.org/es/docs/Web/API/fetch) en su script de componente para hacer peticiones HTTP a APIs usando la URL completa (p.ej. https://example.com/api o `Astro.url + "/api"`).
```

File: src/content/docs/es/guides/view-transitions.mdx [hunk 2]
```diff
@@ -82,20 +63,33 @@ import { ViewTransitions } from 'astro:transitions';
...
+
+Astro asignará automáticamente a los elementos encontrados tanto en la página antigua como en la nueva un `view-transition-name` compartido y único. Esta pareja de elementos coincidentes es inferida por el tipo de elemento y su ubicación en el DOM.
+
+En algunos casos, es posible que desees o necesites identificar estos elementos tú mismo. Puedes especificar un nombre para un par de elementos utilizando la directiva `transition:name`.
+
+```astro
+<aside transition:name="hero">
+```
+
+Nota que el `transition:name` debe ser distinto por elemento. Establece esto manualmente solo cuando Astro no pueda inferir un nombre adecuado por sí mismo, o para un control más fino sobre los elementos coincidentes.
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:82]:
```suggestion
Ten en cuenta que el `transition:name` debe ser distinto por elemento. Establece esto manualmente solo cuando Astro no pueda inferir un nombre adecuado por sí mismo, o para un control más fino sobre los elementos coincidentes.
```

File: src/content/docs/es/guides/view-transitions.mdx [hunk 3]
```diff
@@ -82,20 +63,33 @@ import { ViewTransitions } from 'astro:transitions';
...
 
 ### Directivas de animación integradas
 
-Astro incluye algunas animaciones integradas que puedes agregar con la directiva `transition:animate`: `slide`, `fade` y `morph` (predeterminado).
+Astro incluye con algunas animaciones incorporadas para anular la transción `morph` predeterminada. Agrega la `transición:animate` para probar la animación `slide` o `fade` de Astro.
```
Comment (Waxer59) [inline src/content/docs/es/guides/view-transitions.mdx:86]:
```suggestion
Astro viene con algunas animaciones incorporadas para anular la transción `morph` predeterminada. Agrega la directiva `transición:animate` para probar la animación `slide` o `fade` de Astro.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the nice work peeps 🙌

**Pull request merged**